firebase-database/src/test/java/com/google/firebase/database/MapperTest.java

```diff
@@ -1243,8 +1245,7 @@ public void beanMapsMustHaveStringKeys() {
 
   @Test
   public void serializeStringBean() {
-    StringBean bean = new StringBean();
-    bean.value = "foo";
```

Comment:
Direct member access was disallowed. I had the choice of adding a setValue or just a constructor that accepted a value.